package.json

```diff
@@ -51,7 +49,7 @@
     "moment": "^2.24.0",
     "timsort": "^0.3.0",
     "vis-data": "^6.1.1",
-    "vis-util": "^1.1.2"
+    "vis-util": "^1.1.4"
```

Comment:
dependency update should better be done it a separat PR if it is not related.

Reply:
Done.